test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -2059,4 +2060,82 @@ public void otherSituationsEndCounter(String pref, String fixture) {
+              withTurnsPlayed(111),
+              withFight(0));
+
+      TurnCounter.startCounting(8, "Spooky VHS Tape Monster type=wander", "watch.gif");
+      TurnCounter.startCounting(
+          0, "Spooky VHS Tape unknown monster window begin loc=* type=wander", "lparen.gif");
+      TurnCounter.startCounting(
+          8, "Spooky VHS Tape unknown monster window end loc=* type=wander", "rparen.gif");
+
+      try (cleanups) {
+        String html = html("request/test_fight_spooky_vhs_tape_monster.html");
+        FightRequest.updateCombatData(null, null, html);
+        assertThat("spookyVHSTapeMonster", isSetTo(""));
+        assertThat(TurnCounter.isCounting("Spooky VHS Tape Monster"), is(false));
+        assertThat(
+            TurnCounter.isCounting("Spooky VHS Tape unknown monster window begin"), is(false));
+        assertThat(TurnCounter.isCounting("Spooky VHS Tape unknown monster window end"), is(false));
+      }
+    }
+
+    @Test
+    public void canTrackSpookyVHSTapeSuccess() {
+      var cleanups =
+          new Cleanups(
+              withProperty("spookyVHSTapeMonster"),
+              withProperty("spookyVHSTapeMonsterTurn"),
+              withTurnsPlayed(111),
+              withItem(ItemPool.SPOOKY_VHS_TAPE),
+              withFight(1));
+
+      try (cleanups) {
+        String html = html("request/test_fight_spooky_vhs_tape_success.html");
+        FightRequest.registerRequest(true, "fight.php?action=useitem&whichitem=11270");
+        FightRequest.updateCombatData(null, null, html);
+        assertThat("Spooky VHS Tape", not(isInInventory()));
+        assertThat("spookyVHSTapeMonster", isSetTo("ghost"));
+        assertThat("spookyVHSTapeMonsterTurn", isSetTo(111));
+        assertThat(TurnCounter.isCounting("Spooky VHS Tape Monster"), is(true));
+      }
+
+      TurnCounter.stopCounting("Spooky VHS Tape Monster");
+    }
+
+    @Test
+    public void canTrackSpookyVHSTapeFailure() {
+      var cleanups =
+          new Cleanups(
+              withProperty("spookyVHSTapeMonster"),
+              withProperty("spookyVHSTapeMonsterTurn"),
+              withTurnsPlayed(111),
+              withItem(ItemPool.SPOOKY_VHS_TAPE),
+              withFight(1));
+
+      try (cleanups) {
+        String html = html("request/test_fight_spooky_vhs_tape_failure.html");
+        FightRequest.registerRequest(true, "fight.php?action=useitem&whichitem=11270");
+        FightRequest.updateCombatData(null, null, html);
+        assertThat("Spooky VHS Tape", isInInventory());
+        assertThat("spookyVHSTapeMonster", isSetTo(""));
+        assertThat("spookyVHSTapeMonsterTurn", isSetTo(-1));
+        assertThat(
+            TurnCounter.isCounting("Spooky VHS Tape unknown monster window begin"), is(true));
+        assertThat(TurnCounter.isCounting("Spooky VHS Tape unknown monster window end"), is(true));
+      }
+
+      TurnCounter.stopCounting("Spooky VHS Tape unknown monster window begin");
```

Comment:
Ditto. I'm not sure how much of this there are, so assume I'm commentng on all of them. 😀